src/v/cluster/partition_manager.cc

```diff
@@ -75,9 +75,9 @@ ss::future<consensus_ptr> partition_manager::manage(
   storage::ntp_config ntp_cfg,
   raft::group_id group,
   std::vector<model::broker> initial_nodes,
-  std::optional<cluster::remote_topic_properties> rtp) {
+  std::optional<remote_partition_properties> rpp) {
```

Comment:
Seems that `remote_topic_properties` are only used for recovery, and the bucket only for read replicas. Maybe it is wrong to group them into one struct and they should just be separate parameters?